lldb/source/Target/RegisterContextUnwind.cpp

```diff
@@ -2088,6 +2088,12 @@ bool RegisterContextUnwind::ReadFrameAddress(
     UnwindLogMsg("No suitable CFA found");
     break;
   }
+  case UnwindPlan::Row::FAValue::isConstant: {
+    address = fa.GetConstant();
+    address = abi_sp->FixCodeAddress(address);
```

Comment:
@jasonmolenda  I see a few other places in this file calling the function above (`FixCodeAddress`). But there are many other places that just take addresses and use them as is to do things like `Process:ReadMemory`... it makes me wonder whether we should audit those uses.